core/tag.c

```diff
@@ -153,8 +157,8 @@ size_t lf_readable_time(char* buffer, instant_t time) {
         size_t printed = lf_comma_separated_time(buffer, time / HOURS(1));
         time = time % HOURS(1);
         buffer += printed;
-        snprintf(buffer, 7, " hours");
-        buffer += 6;
+        snprintf(buffer, 4, " hr");
```

Comment:
```suggestion
        snprintf(buffer, 3, " h");
```
https://en.wikipedia.org/wiki/Hour

Reply:
Resolved with commit 6a8c1793d334da97725b96e691e7543778274678.